go-controller/pkg/util/ovs.go

```diff
@@ -169,7 +188,7 @@ func RunOVSVsctl(args ...string) (string, string, error) {
 	cmd.Stderr = stderr
 
 	err = cmd.Run()
-	return strings.Trim(stdout.String(), "\" \n"), stderr.String(), err
```

Comment:
Can you please explain why we are removing the "\n"? Is it because Trimspace is also supposed to remove "\n"?

Reply:
That is true, TrimSpace works on both Windows and Linux. On Windows you have "\r\n" at the end of the line, so it wouldn't strip away the "\r". It's either add the carriage return to the strip chars or change to TrimSpace which does it already.

Reply:
How about using strings.Fields here?

Reply:
It seems that string.Fields splits the string based on the whitespaces and return an array (https://golang.org/pkg/strings/#Fields). Or are you proposing a different approach with string.Fields instead of using strings.Trim?